<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="minimumSize">
           <size>
            <width>541</width>
            <height>459</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QPushButton" name="On_insert">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Вставка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_clear">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Очистка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_delete">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Удаление</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_find">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_print_forward">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Вывести</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Число:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="inBox_2">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>31</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QLabel" name="output">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_MaxDepth">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Наим. предок</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inBox_4">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inBox_3">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
